packages/cli/src/commands/pull.ts

```diff
@@ -30,7 +37,8 @@ const help = () => {
     'DIR'
   )}    Path to the global ${'`.vercel`'} directory
     -d, --debug                    Debug mode [off]
-    --env [filename]               The file to write Development Environment Variables to [.env]
+    --env-file [filename]          The file to write Development Environment Variables to [.env]
+    --environment ${getEnvTargetPlaceholder()} Deployment environment [development]
```

Comment:
```suggestion
    --environment [environment] Deployment environment [development]
```

This one can be changed to just a name. Then lets add examples below with the target placeholder expanded like in `vc env --help`

Reply:
I like it. Now it looks like this:

```
  ▲ vercel pull [path]

 Options:

    -h, --help                     Output usage information
    -A FILE, --local-config=FILE   Path to the local `vercel.json` file
    -Q DIR, --global-config=DIR    Path to the global `.vercel` directory
    -d, --debug                    Debug mode [off]
    --env-file [filename]          The file to write Development Environment Variables to [.env]
    --environment [environment]    Deployment environment [development]
    -y, --yes                      Skip the confirmation prompt

  Examples:

  – Pull the latest Project Settings from the cloud

    $ vercel pull
    $ vercel pull ./path-to-project
    $ vercel pull --env .env.local
    $ vercel pull ./path-to-project --env .env.local

  – Pull specific environment's Project Settings from the cloud

    $ vercel pull --environment=<production | preview | development>
```